ui.content/src/main/content/jcr_root/content/hclecomm/_jcr_content/image/file.dir/.content.xml

```diff
@@ -3,7 +3,7 @@
     jcr:primaryType="nt:file">
     <jcr:content
         jcr:lastModifiedBy="admin"
-        jcr:mimeType="image/png"
+        jcr:mimeType="image/jpeg"
```

Comment:
@DeepaliAgrwl : If you are uploading .png then why this has been changed to jpeg.

Reply:
@rishabh-behl the thumbnail image shared with Ashish was .jpg image. That is why the image mime type is changed.

Reply:
The mime type of hcleocmm-content-page-template thumbanail is also image/jpeg itself